chart/templates/service.yaml

```diff
@@ -8,11 +8,17 @@ spec:
     port: {{ .Values.networking.gamePort | int }}
     targetPort: {{ .Values.networking.gamePort | int }}
     protocol: UDP
+    {{- if eq .Values.networking.serviceType "NodePort" }}
```

Comment:
This seems unrelated to the storage, can you elaborate more on this change please?

Reply:
Hey, sorry for that, I pushed the changes back to the same branch, actually intended to open a second PR for this as you suggested.

This enables setting a specific NodePort.

I am running your chart on bare metal servers without a loadbalancer, so a fixed port (23456) was much nicer for me than an automatically assigned one.

This however has the downside that the user of the chart  has to make sure this port is not already in use by another service (inside or outside of Kubernetes).

Reply:
Makes sense, no harm no foul. Merging.